roq-generator/runtime/src/main/java/io/quarkiverse/roq/generator/runtime/RoqGeneratorConfig.java

```diff
@@ -14,25 +15,33 @@ public interface RoqGeneratorConfig {
 
     /**
      * The selected paths to include in the static website
-     * Glob syntax is authorized to add resources
+     * Glob syntax is authorized for non-dynamic resources (without query or path params)
      * <p>
      * For dynamic paths selection, produce a {@link RoqSelection} in you app.
-     * <p>
-     * <code>
-     *     &#64;Produces
-     *     &#64;Singleton
-     *     &#64;Transactional
-     *     RoqSelection produce() {
-     *         return new RoqSelection(List.of(
-     *                 SelectedPath.builder().html("/roq?name=foo-html").build(),
-     *                 SelectedPath.builder().path("/roq?name=foo").build(),
-     *                 SelectedPath.builder().path("/roq?name=bar").build()));
-     *     }
-     * </code>
+     *
+     * <pre>
+     * {@code
+     * &#64;Produces
+     * &#64;Singleton
+     * @Transactional
```

Comment:
I think `@Transactional` needs to be `&#64;Transactional`

Reply:
yeah it's the other way around, thanks, it's fixed